presto-orc/src/main/java/com/facebook/presto/orc/Filters.java

```diff
@@ -509,6 +507,25 @@ public Filter getMember(SubfieldPath.PathElement member)
         public void addMember(SubfieldPath.PathElement member, Filter filter)
         {
             filters.put(member, filter);
```

Comment:
The following comment might have been lost:

---
Check for duplicates and throw

```
Filter previousFilter = filters.put(member, filter);
checkArgument(previousFilter == null, "Duplicate filter key: " + member);
```